optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/ConstraintCollectors.java

```diff
@@ -68,6 +73,9 @@
     // count
     // ************************************************************************
 
+    private ConstraintCollectors() {
```

Comment:
soft. Code style. Right now it's after the count comment header and before the method. Let's put this constructor after all static methods